bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -11,30 +11,7 @@ import okhttp3.Response
 import social.bigbone.api.Pageable
 import social.bigbone.api.entity.data.InstanceVersion
 import social.bigbone.api.exception.BigBoneRequestException
-import social.bigbone.api.method.AccountMethods
-import social.bigbone.api.method.AppMethods
-import social.bigbone.api.method.BlockMethods
-import social.bigbone.api.method.BookmarkMethods
-import social.bigbone.api.method.ConversationMethods
-import social.bigbone.api.method.DirectoryMethods
-import social.bigbone.api.method.FavouriteMethods
-import social.bigbone.api.method.FeaturedTagsMethods
-import social.bigbone.api.method.FilterMethods
-import social.bigbone.api.method.FollowRequestMethods
-import social.bigbone.api.method.InstanceMethods
-import social.bigbone.api.method.ListMethods
-import social.bigbone.api.method.MarkerMethods
-import social.bigbone.api.method.MediaMethods
-import social.bigbone.api.method.MuteMethods
-import social.bigbone.api.method.NotificationMethods
-import social.bigbone.api.method.OAuthMethods
-import social.bigbone.api.method.PollMethods
-import social.bigbone.api.method.ReportMethods
-import social.bigbone.api.method.SearchMethods
-import social.bigbone.api.method.StatusMethods
-import social.bigbone.api.method.StreamingMethods
-import social.bigbone.api.method.TagMethods
-import social.bigbone.api.method.TimelineMethods
+import social.bigbone.api.method.*
```

Comment:
Could you please replace this star import with explicit imports? Sorry, `gradle check` does not currently report this.